source/CommonJobs/CommonJobs.MVC.UI/Areas/Evaluations/Views/Shared/_Layout.cshtml

```diff
@@ -5,72 +5,61 @@
-                <div class="breadcrum">Dashboard</div>
-            }
-            else if (ViewBag.IsReportDashboard != null && ViewBag.IsReportDashboard)
-            {
-                <div class="breadcrum">HR Dashboard</div>
-            }
-            <div class="pull-right">
-                <form class="form-inline" id="persistenceForm">
-                    <ul class="nav nav-right">
-                        @if (ViewBag.IsCalification != null && ViewBag.IsCalification)
-                        {
-                        <!--ko if: isEvaluationEditable -->
-                        <li><button class="btn btn-primary" disabled="disabled" data-bind="click: onSave, enable: isDirty()">Guardar</button></li>
-                        <!--/ko -->
-                        }
-                        @if (ViewBag.hasAutoCalification != null && ViewBag.hasAutoCalification)
-                        {
-                        <li><a class="btn btn-primary" href="@Url.Content("~/Evaluations/" + ViewBag.Period + "/" + User.Identity.Name)" >AutoEvaluación</a></li>
-                        }
-                        <li>@Html.DocumentationLink("Ayuda")</li>
-                        @if (Request.IsAuthenticated && User != null && User.Identity != null && User.Identity.Name != null) 
-				        {
-                            @*//TODO: We need improve the navigation bar styles. So, I removed these inline styles in this moment*@
-                            <li class="username">
-                                <small>@User.Identity.Name
-                                @if (User.Identity.AuthenticationType == "Forms")
-                                {
-                                    <text>[<a href="@Url.Content("~/Account/LogOff")">LogOff</a>]</text>
-                                }
-                                </small>
-                            </li>
-				        }
-                    </ul>
-                </form>
+<html>
+    <head>
+        <meta charset="utf-8" />
+        <title>@ViewBag.Title</title>
+        @Html.RenderScriptManagerEntries(ScriptManager)
+        <link rel="shortcut icon" href="@Url.Content("~/Content/Images/favicon.ico")" />
+    </head>
+    <body>
+        <header class="header__main">
+            <div class="header__holder">
+                <div class="suit-wrapper">
+                    <a href="@Url.Content("~")" class="logo">
+                        <img src="/Content/Images/Evaluations/header_logo.png" alt="CommonJobs">
+                    </a>
+                    @if (ViewBag.IsUserEvaluator != null && ViewBag.IsUserEvaluator)
+                    {
+                        <div class="header__breadcrum"><a href="@Url.Content("~/Evaluations/" + ViewBag.Period)">Dashboard</a> > Evaluación</div>
+                    }
+                    else if (ViewBag.IsPeriodCreation != null && ViewBag.IsPeriodCreation)
+                    {
+                        <div class="header__breadcrum">Period</div>
```

Comment:
Ah, that makes more sense.

Where is the UI repository? Keep in mind that this is an open sourced project, and so we should make it easy for people to collaborate on it. Having sources minified doesn't help on that, but if we can indicate how those can be modified, then it would solve the problem.

Also, since the source code is for the same project, why was it decided that it would be a different repository? It could have always been a branch or a fork of this same repository.